client/src/components/view.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { useParams } from "react-router";
 import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
This can be done in one line like
`import {useNavigate, Link} from 'react-router-dom';`